docs/introduction.md

```diff
@@ -5,402 +5,103 @@ This document is aimed at introducing developers to the Flow Data Provisioning S
 **Table of Contents**
 
 1. [Getting Started](#getting-started)
-2. [Flow](#flow)
-   1. [Rules](#rules)
-   2. [Errors](#errors)
-   3. [Execution State vs Protocol State](#execution-state-vs-protocol-state)
-3. [Flow Node Roles](#flow-node-roles)
-   1. [Collection Nodes](#collection-nodes)
-      1. [Collection Formation](#collection-formation)
-   2. [Consensus Nodes](#consensus-nodes)
-      1. [Block Formation](#block-formation)
-   3. [Execution Nodes](#execution-nodes)
-      1. [Collection retrieval](#collection-retrieval)
-      2. [Block Execution](#block-execution)
-   4. [Verification Nodes](#verification-nodes)
-   5. [Access Nodes](#access-nodes)
-4. [Glossary](#glossary)
-   1. [Proof of Stake](#proof-of-stake)
-   2. [Staking](#staking)
-   3. [Slashing](#slashing)
-   4. [Sporks](#sporks)
-   5. [Cadence](#cadence)
-   6. [Transactions](#transactions)
-   7. [Byzantine Fault](#byzantine-fault)
-   8. [Merkle Patricia Tries](#merkle-patricia-tries)
-   9. [Specialized Proof of Confidential Knowledge](#specialized-proof-of-confidential-knowledge)
-5. [Developer Guide](#developer-guide)
+   1. [](#)
+2. [Developer Guide](#developer-guide)
    1. [Installation](#installation)
       1. [Dependencies](#dependencies)
-      2. [Manual Build](#manual-build)
+      2. [Build](#build)
    2. [Setting up a test environment](#setting-up-a-test-environment)
-6. [More Resources](#more-resources)
-   1. [Internal](#internal)
-   2. [External](#external)
+3. [More Resources](#more-resources)
 
 ## Getting Started
 
-The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow [execution state](#execution-nodes).
+The Flow Data Provisioning Service (DPS) is a web service that maintains and provides access to the history of the Flow execution state.
 
 The reason for this need is that the in-memory execution state is pruned after 300 chunks, which makes it impossible to access the state history.
-Also, script execution is currently proxied from the [access nodes](#access-nodes) to [execution nodes](#execution-nodes), which is not scalable.
+Also, script execution is currently proxied from the access nodes to execution nodes, which is not scalable.
 The DPS makes access to the execution state _available_ (at any block height) and _scalable_ (so it does not infer load on the network nodes).
```

Comment:
`infer` imo doesn't make sense here. How about `so it does not increase load on the core network`?